tests/v2/validation/resource/terraform/v1charts/aws/variables.tf

```diff
@@ -0,0 +1,135 @@
+variable "rancher_api_url" {
+  type        = string
+  default     = ""
+  description = "Rancher API URL"
+}
+
+variable "rancher_token_key" {
+  type        = string
+  default     = ""
+  description = "Rancher Token Key"
+}
+
+variable "aws_ami" {
+  type        = string
+  default     = ""
+  description = "AWS ami for provisioning instances"
+}
+
+variable "aws_ami_user" {
+  type        = string
+  default     = ""
+  description = "User for the selected ami (used for ssh provisioning)"
+}
+
+variable "aws_subnet" {
+  type        = string
+  default     = ""
+  description = "AWS subnet for instances"
+}
+
+variable "aws_security_groups" {
+  type        = list
+  default     = []
+  description = "List of security group ids"
+}
+
+variable "aws_vpc_id" {
+  type        = string
+  default     = ""
+  description = "AWS VPC ID"
+}
+
+variable "aws_zone" {
+  type        = string
+  default     = ""
+  description = "AWS Zone"
+}
+
+variable "aws_root_size" {
+  type        = string
+  default     = "80"
+  description = "AWS Root Size"
+}
+
+variable "aws_instance_type" {
+  type        = string
+  default     = "t3.xlarge"
+  description = "AWS Instance Type"
+}
+
+variable "aws_volume_type" {
+  type        = string
+  default     = "gp2"
+  description = "AWS Volume Type"
+}
+
+variable "aws_access_key" {
```

Comment:
We may want to add `sensitive = true` to any keys, tokens, passwords or other sensitive inputs and outputs so that they wont be displayed directly in the terraform output. Although even with this enabled the values will still be stored in the tfstate files. This shouldn't be a problem though since this will be wiped away after the job is done AFAIK.